ipcc/src/handle_stub.rs

```diff
@@ -0,0 +1,23 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// Copyright 2023 Oxide Computer Company
+
+use crate::IpccError;
+
```

Comment:
maybe add a comment in here that these stubs are for not-illumos platforms?

Reply:
Fixed in 77bd065.